core/src/test/java/tech/tablesaw/io/csv/CsvReaderTest.java

```diff
@@ -173,6 +173,64 @@ public void testDataTypeDetection() throws IOException {
     assertArrayEquals(bus_types, columnTypes);
   }
 
+  @Test
+  public void testNumberTypeDetectionIgnoreZeroDecimal() throws IOException {
```

Comment:
This test and the next are both testing the same thing. (Same with the two tests with `ignoreZeroDecimal(false)`). It's a bit duplicative to have both, so I think it'd be nicer to have just one of them

Reply:
Thanks. I have removed two tests now (the ones based on `Table.read()`). Please let me know if anything else should be adjusted.